scripts/apps/search/components/fields/translations.tsx

```diff
@@ -10,53 +10,49 @@ interface IProps {
 }
 
 export class Translations extends React.PureComponent<IProps> {
-    originalRef: React.ReactNode;
-    translationsRef: React.ReactNode;
-
     render() {
         const output = [];
 
         if (this.props.item.translated_from != null) {
             output.push(
-                <span key="translated" className="label label--hollow"
+                <button key="translated" className="label label--hollow"
                     onClick={($event) => {
```

Comment:
Why are you using `$` sign in the name? That implies that it'd be a jQuery even, but it isn't.